src/api.rs

```diff
@@ -343,6 +346,11 @@ impl Default for ChromaSamplePosition {
   }
 }
 
+/// Pixel data can be encoded over 8 or 10 (or 12 but unsupported) bits. When data is expressed in:
```

Comment:
why is there a whole sentence about 8, 10, 12 bits? and a whole explanation for each type? also the documentation only needs to convey what this is used for, this chunk of text would work for a help manual

Reply:
It is also incorrect that 12-bit color is unsupported (unless it was broken recently).